sdk/python/packages/flet-core/src/flet_core/types.py

```diff
@@ -31,6 +31,11 @@ class WebRenderer(Enum):
     CANVAS_KIT = "canvaskit"
 
 
+class UrlTarget(Enum):
+    BLANK = "blank"
+    SELF = "_self"
+
```

Comment:
It should be `_blank`. List of possible values: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#target

and an arbitrary string should be allowed as well. It's is "frame name", but I'm not sure if we need that.